packages/paste-website/src/pages/patterns/create/index.mdx

```diff
@@ -0,0 +1,280 @@
+  </Disclosure>
+</Box>
+
+If you find, through usability testing, that the anchor style does not give the create action enough visual prominence, you can use an anchor styled as a button instead. **Only do this in cases where using the anchor style hinders the usability of the feature**, as styling an anchor as a button comes with [accessibility trade-offs](https://marcysutton.com/links-vs-buttons-in-modern-web-applications).
+
+When using an anchor styled as a button, consider adding an arrow icon to offer some affordance that the button will navigate the user to a new page. Also, ensure to add the `as="a"` prop to the button component so that it is rendered as an anchor semantically, while maintaining button styling.
+
+![table with primary action link styled as a button](../../../assets/images/patterns/create-anchor-as-button.png)
+
+<Box marginBottom="space80">
+  <Disclosure variant="contained">
+    <DisclosureHeading as="h5" variant="heading50">
+      Show live example
+    </DisclosureHeading>
+    <DisclosureContent>
+      Coming soon!
+    </DisclosureContent>
+  </Disclosure>
+</Box>
+
+### Sub-section
+
+When a single page includes multiple, separate create actions that are separated by something like tabs, the create action must be placed in the correct hierarchy. Instead of placing the action in the page header, the create action should be placed within the sub-section, such as at the top right of a table within a tab.
+
+![table inside of a tab with a primary action button](../../../assets/images/patterns/create-sub-section.png)
+
+<Box marginBottom="space80">
+  <Disclosure variant="contained">
+    <DisclosureHeading as="h5" variant="heading50">
+      Show live example
+    </DisclosureHeading>
+    <DisclosureContent>
+      Coming soon!
+    </DisclosureContent>
+  </Disclosure>
+</Box>
+
+### Card
+
+When the data being displayed is more suited to a card layout rather than a table layout, the create action will live inside of an empty card at the start of the list of objects.
+
+![list of three cards, the first of which is an empty card with a primary action button](../../../assets/images/patterns/create-card.png)
+
+<Box marginBottom="space80">
+  <Disclosure variant="contained">
+    <DisclosureHeading as="h5" variant="heading50">
+      Show live example
+    </DisclosureHeading>
+    <DisclosureContent>
+      Coming soon!
+    </DisclosureContent>
+  </Disclosure>
+</Box>
+
+### Empty state
+
+If the user has not yet created any instances of this object, they should see an empty state with a create action within it. Follow the [empty state (coming soon!)](/patterns/empty-state) pattern in this case.
+
+## Creation flow
+
+Bringing consistency to what happens after the user triggers the Create action is also a critical part of this pattern.
+
+We've not yet patternized this part of Create. If you need this for your current or upcoming work, please consider partnering with us to [contribute it](/patterns/contribute).
```

Comment:
Would it help to put this part in an actual callout since both Johanna and Ally called it out?

Reply:
Just pushed an update to put this in a callout!